onedal/_device_offload.py

```diff
@@ -94,6 +98,7 @@ def _transfer_to_host(queue, *data):
     host_data = []
     for item in data:
         usm_iface = getattr(item, "__sycl_usm_array_interface__", None)
+        array_api = getattr(item, "__array_namespace__", lambda: None)()
```

Comment:
I think for this PR its fine, but I really would like to see the interactions with array namespace be located in the new ```_array_api.py``` file.

Reply:
Ticket is created 8386